pandas/core/window.py

```diff
@@ -96,6 +96,8 @@ def is_freq_type(self):
         return self.win_type == 'freq'
 
     def validate(self):
+        if self.window == 0:
```

Comment:
As @mroeschke pointed out can't you combine this with the check below? Don't think we need this